package.json

```diff
@@ -102,7 +102,8 @@
   "resolutions": {
     "scrypt.js": "https://registry.npmjs.org/@compound-finance/ethereumjs-wallet/-/ethereumjs-wallet-0.6.3.tgz",
     "ganache-core": "github:compound-finance/ganache-core.git#jflatow/unbreak-fork",
-    "solidity-parser-antlr": "https://github.com/solidity-parser/parser#0.8.2"
+    "solidity-parser-antlr": "https://github.com/solidity-parser/parser#0.8.2",
+    "@venusprotocol/protocol-reserve": "1.3.0-dev.1"
```

Comment:
Is it needed?

Reply:
Yes this is needed because the venus-protocol dependency uses an earlier version of protocol-reserve

Reply:
when venus-protocol updates to use this version or later it can be removed